lib/cover_my_meds/error.rb

```diff
@@ -2,16 +2,20 @@ module CoverMyMeds
   module Error
     class HTTPError < StandardError
 
-      def initialize status, message
+      def initialize status, message, http_method, rest_resource
         @status = status
         @error_json = message
+        @http_method = http_method
+        @rest_resource = rest_resource
       end
 
       def message
-        "#{@status}: #{@error_json}"
+        "#{@status}: #{@error_json}\n"+
+        "in response to:\n"+
+        "#{@http_method.upcase} #{@rest_resource}\n"
```

Comment:
could use a [here doc](http://blog.jayfields.com/2006/12/ruby-multiline-strings-here-doc-or.html) for this